cmd/trace-agent/config/config.go

```diff
@@ -597,3 +594,44 @@ func acquireHostnameFallback(c *config.AgentConfig) error {
 	log.Debugf("Acquired hostname from core agent (%s): %q.", c.DDAgentBin, c.Hostname)
 	return nil
 }
+
+// SetHandler returns handler for runtime configuration changes.
+func SetHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		if req.Method != http.MethodPost {
+			http.NotFoundHandler()
+			return
+		}
+		for set, values := range req.URL.Query() {
```

Comment:
```suggestion
		for key, values := range req.URL.Query() {
```
I think `key` is more appropriate here. It is literally a key that we are setting.